docs/instance.md

```diff
@@ -0,0 +1,37 @@
+Instances
+=========
+
+**TODO** it's just a first draft that needs to be completed and more explicit
+
+Creation
+--------
+
+An instance is created on the command line:
+
+```sh
+$ cozy-stack instances add <domain>
+```
+
+It registers the instance in a global database (**TODO** which one), and
+creates some databases for these doctypes:
+
+- `io.cozy.apps`
+- `io.cozy.manifests`
+- `io.cozy.files`
+- `io.cozy.notifications`
+- `io.cozy.settings`
+
+Then, it creates some folders:
+
+- `/`, with the id `io.cozy.folders-root`
+- `/Apps`, with the id `io.cozy.folders-apps`
```

Comment:
It's a good question and I don't have an answer for the moment.

I'm not sure we will have the locale when the instance is created. Maybe, we will have it only later, when the user fills the registration/onboarding.